src/plugins/intel_cpu/src/nodes/kernels/x64/registers_pool.hpp

```diff
@@ -68,15 +68,15 @@ class RegistersPool {
             return *this;
         }
         Reg(Reg&& other) noexcept : reg(other.reg), regPool(std::move(other.regPool)) {}
-        operator TReg&() {
+        operator TReg&() {  // NOLINT(google-explicit-constructor)
```

Comment:
Please add a reason

Reply:
Also huge diff and XByak::Reg inplicit conversion is used in too many places

Reply:
Also incompatible with XByak API